tdrs-backend/gunicorn.py

```diff
@@ -0,0 +1,24 @@
+"""Gunicorn development config file."""
```

Comment:
```suggestion
"""Gunicorn production config file."""
```

Reply:
Side-topic: for clarity, could we rename this entire file to `gunicorn_prod_cfg.py` and the other to `gunicorn_dev_cfg.py`?

Reply:
for the first round I did not rename it because it needed less work to deploy, but renaming the filenames makes totally sense and should be done.